src/EditorFeatures/Core/Shared/Utilities/IThreadingContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
+{
+    internal interface IThreadingContext
+    {
+        bool HasMainThread
```

Comment:
XML docs please.

Reply:
⌛️ Fixed locally and will be in next commit.